pkg/compose/publish.go

```diff
@@ -76,58 +70,25 @@ func (s *composeService) publish(ctx context.Context, project *types.Project, re
 			return err
 		}
 
-		layer, err := s.pushComposeFile(ctx, "image-digests.yaml", yaml, resolver, named)
-		if err != nil {
-			return err
-		}
-		layers = append(layers, layer)
-	}
-
-	emptyConfig, err := json.Marshal(v1.ImageConfig{})
-	if err != nil {
-		return err
-	}
-	configDescriptor := v1.Descriptor{
-		MediaType: "application/vnd.oci.empty.v1+json",
-		Digest:    digest.FromBytes(emptyConfig),
-		Size:      int64(len(emptyConfig)),
-	}
-	var imageManifest []byte
-	if !s.dryRun {
-		err = resolver.Push(ctx, named, configDescriptor, emptyConfig)
-		if err != nil {
-			return err
-		}
-		imageManifest, err = json.Marshal(v1.Manifest{
-			Versioned:    specs.Versioned{SchemaVersion: 2},
-			MediaType:    v1.MediaTypeImageManifest,
-			ArtifactType: "application/vnd.docker.compose.project",
-			Config:       configDescriptor,
-			Layers:       layers,
-			Annotations: map[string]string{
-				"org.opencontainers.image.created": time.Now().Format(time.RFC3339),
-			},
+		layerDescriptor := ocipush.DescriptorForComposeFile("image-diegests.yaml", yaml)
```

Comment:
Typo ?
```suggestion
		layerDescriptor := ocipush.DescriptorForComposeFile("image-digests.yaml", yaml)
```

Reply:
I blame my cat - fixed in [3b9572b](https://github.com/docker/compose/pull/11239/commits/3b9572b63b0cfa9afe1109f13cc66c0eb14cc232)